docker/safety-rules/key-manager.sh

```diff
@@ -9,6 +9,9 @@ if [ -n "${KEY_MANAGER_CONFIG}" ]; then # Path to key manager config
     echo "${KEY_MANAGER_CONFIG}" > /opt/libra/etc/key_manager.yaml
     params+="--template /opt/libra/etc/key_manager.yaml "
 fi
+if [ -n "${CHAIN_ID}" ]; then
```

Comment:
interesting, why aren't these CFG_?
I guess because Michael hasn't gotten to them, but if you take a look at the rest they are all CFG_...

Reply:
Done, thanks!